.github/pull_request_template.md

```diff
@@ -1,10 +1,13 @@
-<!--
-Thank you for sending the PR! We appreciate you spending the time to work on these changes.
-
-Please make sure that you added appropriate entry in docs/Changelog.md and docs/MigrationGuide.md
-
-You can learn more about contributing to Nussknacker here: https://github.com/TouK/nussknacker/blob/staging/CONTRIBUTING.md
-
-Happy contributing!
-
--->
+## Describe your changes
+
+## Checklist before merge
+- [ ] Code is cleaned from temporary changes and commented lines 
+- [ ] Code formatting checked
+- [ ] Parts of the code that are not easy to understand are documented in the code
+- [ ] Automatic tests added
+- [ ] Showcase in dev-application.conf added if necessary 
+- [ ] Documentation added or updated
+- [ ] Review done                              
+- [ ] All comments resolved
```

Comment:
Not sure how it works now, but maybe we can set restrictions to merge when there are unresolved comments, instead of adding this point?